Minecraft/src/main/java/com/microsoft/Malmo/MissionHandlers/BuildBattleDecoratorImplementation.java

```diff
@@ -65,6 +69,32 @@ public class BuildBattleDecoratorImplementation extends HandlerBase implements I
     private int currentScore = 0;
     private boolean valid = true;
     private boolean initialised = false;
+    // private HashMap<BlockType, BlockType> blockToGhostMap = new HashMap<BlockType, BlockType>();
+
+    private XMLBlockState getGhostVariant(XMLBlockState blockState) {
+        String bt = blockState.getBlock().getLocalizedName();
```

Comment:
You shouldn't use `getLocalizedName` here—otherwise this will all break if I change the language! Use something like `blockState.getBlockType() == BlockType.DIRT` intead.